devtools/dev-requirements_conda.yml

```diff
@@ -39,6 +39,7 @@ dependencies:
   - pytest-benchmark
   - pytest-cov >= 2.6.0
   - pytest-mpl = 0.16.1
```

Comment:
This one's missing `pytest-monitor` that the pip requirements have.

Reply:
good catch